src/components/widget/List/RawList.js

```diff
@@ -60,18 +79,24 @@ class RawList extends Component {
     }
 
     navigate = (up) => {
-        const {selected} = this.state;
-        const {list} = this.props;
+        const {selected, dropdownList} = this.state;
+
+        let selectedIndex = null;
 
-        const next = up ? selected + 1 : selected - 1;
+        dropdownList.map((item, index) => {
+            if(JSON.stringify(item) === JSON.stringify(selected)){
+                return selectedIndex = index;
```

Comment:
and return index of item which is selected to  have possibility get higher or lower index